app/controllers/additional_details/questionnaires_controller.rb

```diff
@@ -26,8 +26,7 @@ def create
   end
 
   def update
-    @questionnaire.update_attributes(questionnaire_params)
-    if @questionnaire.save
+    if @questionnaire.update(questionnaire_params)
```

Comment:
Because .update returns the object, which isn't needed here, it might be better to just use .update_attributes.

http://stackoverflow.com/questions/27684038/rails-paperclip-update-vs-update-attributes